src/Elliptic/Systems/Poisson/FirstOrderSystem.hpp

```diff
@@ -28,11 +28,14 @@ namespace Poisson {
  * first-order PDEs
  *
  * \f[
- * -\frac{1}{\sqrt{\gamma}} \partial_i \sqrt{\gamma}\gamma^{ij} v_j(x) = f(x) \\
+ * -\partial_i \gamma^{ij} v_j(x) - \Gamma^i_{ij}\gamma^{jk}v_k = f(x) \\
  * -\partial_i u(x) + v_i(x) = 0
  * \f]
  *
- * where we have chosen the field gradient as an auxiliary variable \f$v_i\f$.
+ * where we have chosen the field gradient as an auxiliary variable \f$v_i\f$
+ * and where \f$\Gamma^i_{jk}=\frac{1}{2}\gamma^{il}\left(\partial_j\gamma_{kl}
```

Comment:
Just for clarification: you will never be solving for the spatial metric or some function times the spatial metric? Since you have `partial_j gamma_{kl}` my worry is if you need to solve for `f_ij(gamma_{ij})` you then compute the spatial metric from the function you are solving for and then differentiate it. Specifically, if you solve for the conformal factor and `gamma_{ij} = psi^4 \bar{gamma_{ik}}` then this term is no longer a "source" term, I think. Or at least, it's not obvious to me that first derivatives are okay as source terms unless you have an auxiliary variable for `phi_i= partial_i psi` and then compute `partial_i gamma_{kl}` as `4 psi^3 phi_i \bar{gamma}_{kl} + psi^4 \partial_i \bar{gamma}_{kl}`

Reply:
I'm never solving for the conformal metric `\bar{\gamma}` in the XCTS equations; the conformal metric is fixed. All geometric quantities like Christoffel symbols and the Ricci scalar are defined w.r.t the fixed conformal metric in the XCTS equations. Note that this documentation here is for the curved Poisson equation that has no notion of a "conformal" metric, so it doesn't put a `\bar` on the `\gamma`. In this context the metric `\gamma` is the fixed background of the Poisson equation. When calling this code from the XCTS equations the Poisson background-metric `\gamma` is the XCTS conformal metric `\bar{\gamma}`. Was that sufficiently confusing? :)

Reply:
Okay, to be very clear:
In the XCTS system `Gamma` is the Christoffel symbol associated with the conformal metric?

Reply:
Yes.